_rules/html-page-lang-matches-default-ucwvc8.md

```diff
@@ -0,0 +1,278 @@
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This page has a [default language][default page language] of English, which is not the language indicated in the `lang` attribute.
+
+```html
+<html lang="da">
+	<head>
+		<title>ACT Rules Format 1.0 - Abstract</title>
+	</head>
+	<body>
+		<p>
+			The Accessibility Conformance Testing (ACT) Rules Format 1.0 defines a format for writing accessibility test
+			rules. These test rules can be used for developing automated testing tools and manual testing methodologies. It
+			provides a common format that allows any party involved in accessibility testing to document and share their
+			testing procedures in a robust and understandable manner. This enables transparency and harmonization of testing
+			methods, including methods implemented by accessibility test tools.
+		</p>
+	</body>
+</html>
+```
+
+#### Failed Example 2
+
+This page has a [default language][default page language] of English because most words are English. This does not match the language indicated in the `lang` attribute.
+
+```html
+<html lang="nl">
+	<head>
+		<title>Gelukkig</title>
+	</head>
+	<body>
+		<p>The Dutch word "gelukkig" has no equivalent in English.</p>
+	</body>
+</html>
+```
+
+#### Failed Example 3
+
+This page has a [default language][default page language] of Dutch because all English text is wrapped in a `p` element with a `lang` attribute set to `en`. Dutch ("nl") is not set as the language of the page by the `lang` attribute on the `html` element.
+
+```html
+<html lang="en">
+	<head>
+		<title>Met de kippen op stok</title>
+	</head>
+	<body>
+		<blockquote>
+			<p>"Hij ging met de kippen op stok"</p>
+		</blockquote>
+		<p lang="en">
+			This Dutch phrase literally translates into "He went to roost with the chickens", but it means that he went to bed
+			early.
+		</p>
+	</body>
+</html>
+```
+
+#### Failed Example 4
+
+This page has a [default language][default page language] of English because the `title` element and the [accessible name][] of the `img` element are English. English ("en") is not set as the language of the page by the `lang` attribute on the `html` element.
+
+```html
+<html lang="nl">
+	<head>
+		<title>Fireworks over Paris</title>
+	</head>
+	<body>
+		<img src="/test-assets/shared/fireworks.jpg" alt="Fireworks over Paris" />
+		<p lang="nl">
+			Gelukkig nieuwjaar!
+		</p>
+	</body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This is an SVG [document][document element], not an HTML document.
+
+```svg
+<svg xmlns="http://www.w3.org/2000/svg" lang="fr"></svg>
+```
+
+#### Inapplicable Example 2
+
+This page has an undefined [default language][default page language] because it has no content or [document title][].
+
+```html
+<html></html>
+```
+
+#### Inapplicable Example 3
+
+This page has an undefined [default language][default page language] because it has no [document title][] all its content is wrapped in element with a `lang` attribute.
```

Comment:
```suggestion
This page has an undefined [default language][default page language] because it has no [document title][] and all its content is wrapped in an element with a `lang` attribute.
```